codelabs/solace-pubsub-microgateway/codelab.json

```diff
@@ -0,0 +1,29 @@
+{
+  "environment": "web",
+  "format": "html",
+  "prefix": "https://storage.googleapis.com",
+  "mainga": "UA-49880327-14",
```

Comment:
Update to match GA ID in tutorial (UA-3921398-10)

Reply:
I have no idea what "mainga" is, or where that number came from?  In my Markdown, I have: "analytics account: UA-3921398-10".  Do I have to manually change the claat generated JSON file?

Reply:
Sent you a screenshot in Slack.  There are 2 separate tracking IDs.

Reply:
Yea, I believe it's google's GA ID for codelabs. They probably just want to know everywhere it's being used and it's added in during generation of the static content by claat so not your fault. I was just considering replacing it with our GA ID when we merge it in......I guess there are pros/cons either way. Google would be tracking us, but they would also then see that codelabs is being used more...

Reply:
As discussed, leaving Google's ID alone.